IAMOTZ/ievents
# #152797308 Implement the signup page with react

#### What does this PR do?
It uses react and redux to implement the sign-up page consuming the API endpoint for creating users.
#### Description of Task to be completed?
User should be able to signup on the application
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started, the signup page should be loaded there
- Fill the form and click register, the data returned by the server should log in the console
#### Any background context you want to provide?
- The frontend still uses the locally hosted server, so you would have to create a .env file to fulfill all the needed environment variables, the run `npm run server:dev`
#### What are the relevant pivotal tracker stories?
#152797308
#### Screenshots (if appropriate)
![screencapture-localhost-8080-1511960399721 1](https://user-images.githubusercontent.com/26048536/33376394-cf46433a-d50d-11e7-9936-b5055f1dc948.png)
#### Questions:
None
